pkg/config/config.go

```diff
@@ -137,7 +137,7 @@ func SetupConfigFolder() error {
 		return err
 	}
 	if _, err := os.Stat(grantedFolder); os.IsNotExist(err) {
-		err := os.Mkdir(grantedFolder, 0700)
+		err := os.Mkdir(grantedFolder, 0644)
```

Comment:
@meyerjrr for all of these, I think it s would be good to add them as a const in the file where they are used with the comment on read/Write

for any files that have the constant but it is unused, remove it

```suggestion
		err := os.Mkdir(grantedFolder, USER_READ_WRITE_PERM)
```

Reply:
If we have other file perms in use, you coud follow the same pattern of adding them as consts